client/src/components/Preview/Preview.js

```diff
@@ -130,11 +136,16 @@ class Preview extends Component {
 }
 
 Preview.propTypes = {
+  extraClass: React.PropTypes.string,
```

Comment:
Why do we need extraClass if we have className property? It's a duplicate of the same feature.

Preview.js isn't a form field so it shouldn't need to match the formfield conventions.